src/components/learner-credit-management/MemberDetailsTableCell.jsx

```diff
@@ -0,0 +1,84 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {
+  IconButton,
+  Stack,
+  Icon,
+} from '@edx/paragon';
+import {
+  Person,
+} from '@edx/paragon/icons';
+
+const MemberDetailsTableCell = ({
+  row,
+}) => {
+  let memberDetails;
+  let memberDetailIcon = (
+    <IconButton
+      isActive
+      invertColors
+      src={Person}
+      iconAs={Icon}
+      className="border rounded-circle mr-3"
+      alt="members detail column icon"
+    />
+  );
+  if (row.original.status === 'removed') {
+    memberDetails = (
+      <div className="mb-n3">
+        <p className="text-danger-900 font-weight-bold mb-0">
```

Comment:
Besides the color should be the same formatting as this line in the assignments table https://github.com/openedx/frontend-app-admin-portal/blob/8e145f93780545ed352fe6f091e677b84c46fe6d/src/components/learner-credit-management/SpendTableEnrollmentDetails.jsx#L20